andela/metis-ah
# #159987405 users should be able to create articles

**What does this PR do?**
Implement create article functionality

**Description of Task to be completed?**
- install `formidable`
- install `cloudinary`
- install `slug`
- install `uuid-random`
- setup `cloudinary`
- write tests for the endpoint
- write a validator middleware

**How should this be manually tested?**
Clone the repository.
Checkout the `ft-Users-should-be-able-to-create-articles-159987405` branch.
Run `npm install` on your local machine.
Run database migrations using `npm run migrate`.
Start the server using `npm start`.

**You can test the following endpoints using postman**
POST `localhost:8000/api/v1/articles`
HEADERS
- authorization -> validjwttoken342klskaus.fdkslie32
- Content-Type -> application/x-www-form-urlencoded

NOTE: 
    Ensure to use `form-data` panel when inputting article data

REQUIRED FIELDS
-    title : `string`
-   description : `string`
-   body : `string`
-   image (optional) : `file`


**What are the relevant pivotal tracker stories?**
#159987405
